src/lib/dom-api.html

```diff
@@ -593,11 +593,73 @@
 
     };
 
+    Object.defineProperties(DomApi.prototype, {
+      activeElement: {
+        configurable: true,
+        get: function() {
+          var ownerDocument = this.node.ownerDocument;
+
+          if (ownerDocument || !window.ShadowDOMPolyfill) {
+            return this.node.activeElement;
+          } else {
+            return window.ShadowDOMPolyfill.wrapIfNeeded(this.node).activeElement;
+          }
+        }
+      }
+    });
+
     // changes and accessors...
     if (!Settings.useShadow) {
 
       Object.defineProperties(DomApi.prototype, {
 
+        activeElement: {
+          get: function() {
+            var active = document.activeElement;
```

Comment:
Generally we don't space-ify within functions and always include `{}` around conditionals.